### DISCUSSION TITLE
API version 4

### DISCUSSION THREAD

**Discussion by losolio** [losol/eventuras#909] [Ideas]
- [ ] Move organization id from request param to request header

**Comment by yura-b** [comment]
> Move organization id from request param to request header

Also an alternative - move organization id to user claims (jwt token), so organization id is always part of authenticated user

**Comment by losolio** [comment]
@yura-b  Thanks for the suggestion, but both users and admins might have permissions and associations with multiple organisations, and will in the future be allowed to switch org/tenant through the ui.

**Comment by losolio** [comment]
converting this to an discussion... not likely to happen soon
